### ISSUE TITLE
Reports Ubuntu Xenial as unsupported during install

### ISSUE THREAD

**Issue by sptramer** [Azure/autorest#2615]
When running `autorest --reset` to reset local configuration issues, I ran into an error message regarding Ubuntu 16.04 LTS (Xenial) indicating that it was not supported:

```
 ∅ ⤅  % autorest --reset
AutoRest Code Generator
(c) 2017 Microsoft Corporation. https://aka.ms/autorest
Error: Unsupported Linux distro: ubuntu, 16.04, unknown
    at Function.getRuntimeId (/usr/local/lib/node_modules/autorest/utility/platform.js:210:23)
    at new PlatformInformation (/usr/local/lib/node_modules/autorest/utility/platform.js:85:50)
    at GetCurrent.Promise.all.then (/usr/local/lib/node_modules/autorest/utility/platform.js:139:20)
    at <anonymous>
Unable to find framework for null
```

Presumably, Xenial is supported at the same level as Wheezy, but is just not included in the compatibility list.

**Comment by sptramer** [comment]
Worth noting that this issue did not appear the __second__ time I ran `--reset` after a botched install (due to another local configuration issue -- invoked from a directory removed during the delete/install process.) This might have affected this bug report and resulted in the autorest runtime not being appropriately downloaded.

**Comment by fearthecowboy** [comment]
try `npm install -g autorest@preview` -- it has better support for runtimes

we're promoting that to stable today.

**Comment by fearthecowboy** [comment]
We've published `2.0.4143` and marked it stable. (requires node 7.10.0 or greater!)

**Comment by fearthecowboy** [comment]
Closing this, since the 1.2.2 build is now deprecated.
